todo_app/tests/test_trello_calls.py

```diff
@@ -0,0 +1,51 @@
+import os
+import pytest
+import requests
+from dotenv import load_dotenv, find_dotenv
+from todo_app import app
+
+
+class StubResponse():
+    def __init__(self, fake_response_data):
+        self.fake_response_data = fake_response_data
+
+    def json(self):
+        return self.fake_response_data
+
+
+def stub(request_string, params={}):
+    url = request_string.split('?')[0]
```

Comment:
see above, this is where the params object comes in handy (i don't think url will ever contain a ?)

Reply:
Maybe, but for the way the app communicates, it was a necessary parse (see the build_url function works)